### ISSUE TITLE
General Questions

### ISSUE THREAD

**Issue by Platygator** [xRadne/TME290#1]
P. 29: Why does send has {} and receive ()?
What are the empty [] for?

**Comment by xRadne** [comment]
Hi Platygator! 
I am an absolute beginner on c++, I literally started 4 hours ago by following a totorial from a teacher at my University.

Therefore I am unable to answer your question. How come you found this repository?

**Comment by Platygator** [comment]
I should have specified that I am Jan. Sorry Alex, don't know why I choose this name back then and no clue where the Avatarpicture comes from 😄 
But I just wanted to collect questions somewhere so we can check those when having a meeting, maybe the Wiki is actually the better location for this.

**Comment by xRadne** [comment]
Haha I expected that this was someone from our class 😆
This is fine with me!

**Comment by xRadne** [comment]
### To discuss:

> What are the empty [] for?    

Explained in lecture 2b at time 02:15:00. "Capture" syntax

### Alex notes:
What is `.odvd`?

**Comment by Platygator** [comment]
Yeah, I needed further details on that, his explanation was a little brief
